Cargo.toml

```diff
@@ -33,12 +34,13 @@ serde = { workspace = true }
 serde_json = { workspace = true }
 anyhow = "1.0.71"
 bytes = { workspace = true }
-signal-hook = "0.3.15"
 tiny_http = "0.12.0"
 ffmpeg-next = "6.0.0"
 crossbeam-channel = { workspace = true }
 env_logger = { workspace = true }
-log = { workspace = true } 
+log = { workspace = true }
+signal-hook = { workspace = true }
 
 [dev-dependencies]
 reqwest = { workspace = true }
+signal-hook = { workspace = true }
```

Comment:
if sth is in dependencies already, does it need to be in dev dependencies too?

Reply:
It's not needed. I forgot to remove it 😅